frontend/src/views/WheresMyGene/components/GeneSearchBar/components/ShareButton/utils.tsx

```diff
@@ -10,18 +10,20 @@ import {
 
 export const generateAndCopyShareUrl = (
   filters: State["selectedFilters"],
+  organism: State["selectedOrganismId"],
   tissues: State["selectedTissues"],
   genes: State["selectedGenes"]
 ) => {
   // Create a URL that contains the selected filters, tissues, and genes as params in the URL
   // This URL can be shared with others to reproduce the same view
   const url = new URL(window.location.href);
+  url.searchParams.set("organism", organism ?? "");
   Object.entries(stripEmptyFilters(filters)).forEach(([key, value]) => {
     url.searchParams.set(key, value.join(","));
   });
   url.searchParams.set("tissues", tissues.join(","));
   url.searchParams.set("genes", genes.join(","));
-  url.searchParams.set("ver", "2");
+  url.searchParams.set("ver", "3");
```

Comment:
why are we bumping the version here? Is it because we're changing `delimiter` from `version === "1" ? "-" : ","` to `version > "1" ? "," : "_"`

Reply:
no version bump needed!